google/cloud/firestore_v1/base_aggregation.py

```diff
@@ -230,7 +237,12 @@ def get(
             retries.Retry, None, gapic_v1.method._MethodDefault
         ] = gapic_v1.method.DEFAULT,
         timeout: float | None = None,
-    ) -> List[AggregationResult] | Coroutine[Any, Any, List[AggregationResult]]:
+        *,
+        explain_options: Optional[ExplainOptions] = None,
+    ) -> (
+        QueryResultsList[AggregationResult]
+        | Coroutine[Any, Any, List[AggregationResult]]
```

Comment:
Shouldn't the Coroutine also return QueryResultsList? (Also, `Awaitable[` seems cleaner than `Coroutine[Any, Any`

Reply:
I didn't change this part because async isn't implemented in this PR. I will change it to use `Awaitable[` instead.